definitions/install.yaml

```diff
@@ -762,6 +762,10 @@ spec:
         env:
         - name: INIT_CONFIG
           value: init-config
+        - name: KYVERNO_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
```

Comment:
Also need to change the resources that are deployed to kyverno namespace, [clusterRoleBindings](https://github.com/nirmata/kyverno/pull/955/files#diff-2e6704d285c61f6d003b8e63a115c2d1L704), [service](https://github.com/nirmata/kyverno/pull/955/files#diff-2e6704d285c61f6d003b8e63a115c2d1R733), [configMap](https://github.com/nirmata/kyverno/pull/955/files#diff-2e6704d285c61f6d003b8e63a115c2d1R725), Namespace, etc.